src/simsopt/util/famus_helpers.py

```diff
@@ -0,0 +1,548 @@
+"""
+    This file contains a class for reading in FOCUS-style data files. 
+
+    This file was copied over from the MAGPIE code used for generating 
+    permanent magnet structures. All credit for this code is thanks 
+    to the PM4Stell team and Ken Hammond for his consent to use this file
+    and work with the permanent magnet branch of SIMSOPT.
+"""
+__all__ = ['FocusData', 'FocusPlasmaBnormal', 'stell_point_transform', 'stell_vector_transform']
+import numpy as np
+import sys
+from simsopt.geo import Surface
+
+FOCUS_PLASMAFILE_NHEADER_TOP = 1
+FOCUS_PLASMAFILE_NHEADER_BDRY = 2
+FOCUS_PLASMAFILE_NHEADER_BNORM = 2
+
+
+class FocusPlasmaBnormal(object):
+
+    def __init__(self, fname):
+
+        with open(fname, 'r') as f:
+            lines = f.readlines()
+            f.close()
```

Comment:
Line 25 is unnecessary. When exiting the `with` block, the file is closed automatically.